config/manager/kustomization.yaml

```diff
@@ -4,8 +4,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: antrea-operator
-  newName: antrea-operator
-  newTag: latest
+  newName: antrea/antrea-operator
+  newTag: 0.0.1
 - name: caorui/antrea-operator
```

Comment:
nit: at some point your dockerhub's entry might be removed. Not sure if you still need it.

Reply:
Rename caorui/antrea-operator to antrea/antrea-operator.